### PR TITLE
Add T36 IT CPE conditions

### PR DESCRIPTION
**Pull request by tvami**
#### PR description:

Add T36 IT CPE conditions

```
SiPixelSimLorentzAngle_phase2_IT_v7.4.1_25x100_v1_mc
SiPixelLorentzAngle_phase2_IT_v7.4.1_25x100_v1_mc
SiPixelTemplateDBObject_phase2_IT_v7.4.1_25x100_v1_mc
SiPixelGenErrorDBObject_phase2_IT_v7.4.1_25x100_v1_mc
SiPixelLorentzAngle_phase2_IT_v7.4.1_25x100_empty_mc
```
 
LA PI https://cern.ch/sfb6s
GenErr/Template headers: https://cern.ch/gxdt5
For the empty LA: https://cern.ch/qlk27

### CODE REVIEW

File: Configuration/AlCa/python/autoCondPhase2.py [hunk 1]
```diff
@@ -29,15 +29,15 @@
...
 }
 
 allTags["LAWidth"] = {
     'T21' : ( ','.join( [ 'SiPixelLorentzAngle_phase2_T15_mc_forWidthEmpty' ,SiPixelLARecord,connectionString, "forWidth", "2019-11-05 20:00:00.000"] ), ),  # uH=0.0/T (fall-back to offset)
     'T25' : ( ','.join( [ 'SiPixelLorentzAngle_phase2_T19_mc_forWidthEmpty' ,SiPixelLARecord,connectionString, "forWidth", "2020-02-23 14:00:00.000"] ), ),  # uH=0.0/T (fall-back to offset)
     'T30' : ( ','.join( [ 'SiPixelLorentzAngle_phase2_IT_v6.4.0_25x100_empty_mc' ,SiPixelLARecord,connectionString, "forWidth", "2021-11-29 20:00:00.000"] ), ),  # uH=0.0/T (fall-back to offset)
     'T33' : ( ','.join( [ 'SiPixelLorentzAngle_phase2_IT_v7.1.1_25x100_empty_mc' ,SiPixelLARecord,connectionString, "forWidth", "2023-12-02 15:55:00.000"] ), ),  # uH=0.0/T (fall-back to offset)
-    'T36' : ( ','.join( [ 'SiPixelLorentzAngle_phase2_IT_v7.4.1_25x100_empty_mc' ,SiPixelLARecord,connectionString, "forWidth", "2024-??-?? ??:00:00.???"] ), ),  # uH=0.0/T (fall-back to offset)
+    'T36' : ( ','.join( [ 'SiPixelLorentzAngle_phase2_IT_v7.4.1_25x100_empty_mc' ,SiPixelLARecord,connectionString, "forWidth", "2024-03-04 21:00:00.000"] ), ),  # uH=0.0/T (fall-back to offset)
```
Comment (mmusich) [inline Configuration/AlCa/python/autoCondPhase2.py:40]: Hi @tvami, this doens't exist yet in the production DB: https://cms-conddb.cern.ch/cmsDbBrowser/search/Prod/SiPixelLorentzAngle_phase2_IT_v7.4.1_25x100_empty_mc

Comment (tvami) [reply Configuration/AlCa/python/autoCondPhase2.py:40]: Thanks Marco, it's uploaded now! All zeros: https://cern.ch/qlk27

**Comment by tvami** [comment]
hi @Raffaella07 please merge this PR, thanks!

File: Configuration/AlCa/python/autoCondPhase2.py [hunk 2]
```diff
@@ -29,15 +29,15 @@
...
 }
 
 allTags["LAfromAlignment"] = {
```
Comment (mmusich) [inline Configuration/AlCa/python/autoCondPhase2.py:43]: @tvami wasn't this dictionary forgotten?

Comment (tvami) [reply Configuration/AlCa/python/autoCondPhase2.py:43]: Yes, sorry,  I did it in https://github.com/Raffaella07/cmssw/pull/7

**Comment by mmusich** [comment]
@tvami I have a couple of questions:
   1. why do we need new CPE payloads in the first place ?
   
`SiPixelLorentzAngle_phase2_IT_v7.4.1_25x100_v1_mc` (for T36, presumably also for T37 and T38)  is the exactly the same payload as the one for T33 `SiPixelLorentzAngle_phase2_IT_v7.1.1_25x100_v1_mc` (including the hash):

```
$ conddb list SiPixelLorentzAngle_phase2_IT_v7.1.1_25x100_v1_mc

Since: Run   Insertion Time       Payload                                   Object Type           
-----------  -------------------  ----------------------------------------  -------------------   
1            2023-05-15 20:54:50  36b60290feab9ee63710ec47ae66426798d9c712  SiPixelLorentzAngle   
 
$ conddb list SiPixelLorentzAngle_phase2_IT_v7.4.1_25x100_v1_mc

Since: Run   Insertion Time       Payload                                   Object Type           
-----------  -------------------  ----------------------------------------  -------------------   
1            2024-03-04 20:37:49  36b60290feab9ee63710ec47ae66426798d9c712  SiPixelLorentzAngle   
```
 
image

2. the header of `SiPixelTemplateDBObject_phase2_IT_v7.4.1_25x100_v1_mc` (but I guess also for the other template-derived CPE conditions) shows that we have a template (with ID 411) that has B-field =0 ( I presume this is the template for the Layer 1 3D pixels).

image

This generates warnings at runtime or the type:

```
%MSG-w UnexpectedMagneticFieldUsingNonIdealPixelTemplate:   SiPixelTemplateDBObjectESProducer:siPixelTemplateDBObjectESProducer@callESModule  13-Mar-2024 21:35:54 CET PostStreamBeginLumi
Magnetic field is 3.8112 Template Magnetic field is 0
%MSG
```

This is actually true also for geometry T35 (see this comment): https://github.com/cms-sw/cmssw/pull/43903#issuecomment-1935639667

any chance you can fix it?

FYI @Raffaella07 @emiglior

**Comment by tvami** [comment]
> why do we need new CPE payloads in the first place ?

I was told by @Raffaella07 @emiglior that the new geometry includes more/different DetIDs so they requested it and I did it with the new branch with the new geometry. Based on what you see with the hashes, this was not neccessary.

> any chance you can fix it?

Not really, the 3D sensors have zero magnetic field in them so that's a feature. Hacking the input templates could be a solution but I'm not sure that would lead to issues at a different part. Anyway I brought this up with Morris a year ago or so and he said this is how the 3D sensor templates should be

**Comment by mmusich** [comment]
> Hacking the input templates could be a solution but I'm not sure that would lead to issues at a different part. 

I disagree.
For T25 (which is also using 3D sensors) it was done this way 

image

and I don't think it has lead to any issue when run in relvals.

>  Anyway I brought this up with Morris a year ago or so and he said this is how the 3D sensor templates should be

well, in that case Morris or whomever is responsible of the pixel local reconstruction should fix the code at 

https://github.com/cms-sw/cmssw/blob/4a6d7e831d09c01a92d2381c319f1424a524e651/CalibTracker/SiPixelESProducers/plugins/SiPixelTemplateDBObjectESProducer.cc#L69-L71

**Pull request merged**